cloud/gcp-functions/cmd/orphanedResources.go

```diff
@@ -0,0 +1,81 @@
+package orphanedResources
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"google.golang.org/api/compute/v1"
+)
+
+func CleanOrphanedResources(w http.ResponseWriter, r *http.Request) {
+
+	ctx := context.Background()
+	computeService, err := compute.NewService(ctx)
+	if err != nil {
+		log.Fatal("Error: create service", err)
+	}
+
+	project := os.Getenv("GCP_DEV_PROJECT")
+	targetPoolAggregatedList, err := computeService.TargetPools.AggregatedList(project).Do()
+
+	if err != nil {
+		log.Fatal("Error: get targetPoolAggregatedList", err)
+	}
+
+	toDelete := false
+	for _, targetPoolList := range targetPoolAggregatedList.Items {
+		for _, targetPoolItem := range targetPoolList.TargetPools {
+			zone := strings.Split(targetPoolItem.Instances[0], "/")[8]
+			instanceName := strings.Split(targetPoolItem.Instances[0], "/")[10]
```

Comment:
if `targetPoolItem.Instances` is empty, this line will panic in runtime. you need to check and handle it before accesing the first element